webapp/channels/src/sass/components/_suggestion-list.scss

```diff
@@ -115,14 +115,17 @@
 }
 
 .slash-command__info {
+    max-width: 89%;
```

Comment:
I'm not sure this `max-width` is necessary—I think `.slash-command__desc`'s `white-space: break-spaces;` should hande things alright

Reply:
Yes, it wasn't necessary. I removed it and pushed a new commit.